xml/decoders/SoundTraxx_Tsu2_Diesel.xml

```diff
@@ -0,0 +1,1508 @@
+                <label>F12 controls output Brake</label>
+        </variable>
+        <variable item="F12 controls output 15" CV="46" mask="VXXXXXXX" default="0" minFn="12">
+                <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+                <label>F12 controls output Coupler</label>
+        </variable>
+      <variable CV="112" mask="XXXXXXXV" item="Sound Option 14" default="1">
+        <enumVal>
+          <enumChoice>
+            <choice>Emergency stop engine shutdown</choice>
+          </enumChoice>
+          <enumChoice>
+            <choice>Emergency stop engine idle</choice>
+          </enumChoice>
+        </enumVal>
+        <label>Emergency Stop</label>
+      </variable>
+      <variable CV="112" mask="XXXXXXVX" item="Sound Option 15" default="1">
+	<xi:include href="http://jmri.org/xml/decoders/parts/enum-disabledEnabled.xml"/>
+        <label>Low-pressure alarm bell</label>
+      </variable>
+      <variable CV="114" mask="XXXXVVVV" item="Sound Option 1" default="7" tooltip="Controls how the engine RPMs interact with the speed step setting of the throttle">
+        <xi:include href="http://jmri.org/xml/decoders/soundtraxx/TSUenumNotch.xml"/>
+        <label>Engine Notch Rate</label>
+      </variable>
+      <variable CV="114" mask="XXXVXXXX" item="Sound Option 2" default="0" tooltip="When this option AND manual (notching) engine control is enabled, locomotive will not move until engine starts">
+        <xi:include href="http://jmri.org/xml/decoders/soundtraxx/TSUenumInterlock.xml"/>
+        <label>Engine RPM Interlock</label>
+      </variable>
+      <variable CV="114" mask="XXVXXXXX" item="Sound Option 3" default="1" tooltip="Determines whether prime mover will start up automatically or manually">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-DisableEnable_01.xml"/>
+        <label>Engine Auto Start Enable</label>
+      </variable>
+      <variable CV="114" mask="VVXXXXXX" item="Sound Option 4" default="0" tooltip="Controls how the dynamic brakes interact with the engine RPMs">
+        <xi:include href="http://jmri.org/xml/decoders/soundtraxx/TSUenumDynamic.xml"/>
+        <label>Dynamic Brake Override</label>
+      </variable>
+      <variable CV="116" mask="XVVVVVVV" item="Advanced Group 1 Option 8">
+          <decVal max="127"/>
+          <label>Dynamic Brake Rate</label>
+      </variable>
+      <variable CV="116" mask="VXXXXXXX" item="Advanced Group 1 Option 9">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-AddSubtract2.xml"/>
+          <label>Dynamic Brake Rate Sign</label>
+      </variable>
+      <!-- Sound Selection follows -->
+      <variable CV="120" mask="XXVVVVVV" item="Sound Option 6" default="0" tooltip="Selects which airhorn to use">
+        <enumVal>
+          <xi:include href="http://jmri.org/xml/decoders/soundtraxx/TSU2enumChoiceDieselAirhorn.xml"/>
```

Comment:
This should be `<xi:include href="http://jmri.org/xml/decoders/soundtraxx/TSU2enumChoiceDieselAirHorn.xml"/>` to match the filename exactly.

I'm pretty sure that's why it fails on Travis (it's a Linux box with a case-sensitive filesystem) whereas AppVeyor is a Windows box with a case insensitive file system.